cmd/oras/root/cp.go

```diff
@@ -265,3 +237,50 @@ func recursiveCopy(ctx context.Context, src oras.ReadOnlyGraphTarget, dst oras.T
 	}
 	return err
 }
+
+func prepareCopyOption(ctx context.Context, src oras.ReadOnlyGraphTarget, dst oras.Target, root ocispec.Descriptor, opts *oras.ExtendedCopyOptions) error {
```

Comment:
It would be better to have this pattern
```suggestion
func prepareCopyOption(ctx context.Context, src oras.ReadOnlyGraphTarget, dst oras.Target, root ocispec.Descriptor, opts oras.ExtendedCopyOptions) (oras.ExtendedCopyOptions, error) {
```

Reply:
Good catch.